content/en/docs/getting-started/_index.html

```diff
@@ -5,7 +5,8 @@
 
 aliases = [
     "/getting-started/",
-    "/docs/getting-started"
+    "/docs/getting-started",
```

Comment:
Is this really needed? That's really just a redirect to itself, right? Luckily it seems to get ignored. @ethangj have you tried without?